src/core/library.c

```diff
@@ -295,6 +341,13 @@ bool _mLibraryAddEntry(struct mLibrary* library, const char* filename, const cha
 	core->getGameCode(core, entry.internalCode);
 	core->checksum(core, &entry.crc32, mCHECKSUM_CRC32);
 	entry.platform = core->platform(core);
+	entry.platformModels = GB_MODEL_AUTODETECT;
+	if (entry.platform == mPLATFORM_GB) {
```

Comment:
That implies that I need to create a distinct constant for "unknown model" as separate from "autodetect" so that we can be sure to set it to a known, distinct value if `M_CORE_GB` isn't enabled. That'll make sure that there's a clear indication in the database that it needs rescanned if you keep the same library database with different builds.